Command/FixMediaContextCommand.php

```diff
@@ -32,9 +32,17 @@ public function configure()
      */
     public function execute(InputInterface $input, OutputInterface $output)
     {
+        if (false === $this->getContainer()->has('sonata.media.manager.category')) {
```

Comment:
Use `!$this->getContainer()->has('sonata.media.manager.category')`. This method will always return a boolean.